language/move-prover/lab/README.md

```diff
@@ -0,0 +1,35 @@
+This crate contains tools and data for analyzing the performance of the Move Prover.
+
+The `data` directory contains a number of lab sessions with scripts and persisted data. Each of those directories
+has its own README. The `src` directory contains supporting Rust code.
+
+Most labs are currently based on Jupyter notebooks with a Rust based kernel. The installation of Jupyter
+can be done via the python installation tool:
+
+```
+pip install jupyterlab
+pip install notebook
+```
+
+For the Rust kernel, we currently need to use a
+[not-yet-released version](https://github.com/google/evcxr/issues/161#issuecomment-815660992). Install
```

Comment:
For some reason, my local install of evcxr doesn't have this issue.

Reply:
So at least we know that `cargo install evcxr_jupyter --version 0.6.0` should work. (It may have other issues compared to the more recent version on the main branch, though.)

Reply:
It seems to be a bug recently introduced.